public/js/index.js

```diff
@@ -484,9 +498,21 @@ Index.handleContextMenu = function (option, id) {
         LRR.openInNewTab(`./edit?id=${id}`);
         break;
     case "delete":
-        if (window.confirm("Are you sure you want to delete this archive?")) {
-            Server.deleteArchive(id, () => { document.location.reload(true); });
-        }
+        window.Swal.fire({
```

Comment:
Similarly, swal2 popups should be themed (example for modern.css)  

```
.swal2-popup {
  background: #43464E !important;
  color: #DDDDDD !important;
}
```

![image](https://user-images.githubusercontent.com/8237712/166220759-33c4e8b1-a4a1-455d-8edd-dd534a0df7f5.png)

Reply:
Needs colors for the other themes

Reply:
Actually .swal2-popup is themed. The color is careful picked from the theme and then modified by adding more light. It is meant to give enough transition from the background and the pop-up window.
https://github.com/Difegue/LANraragi/pull/620/files#diff-f2a0f5f9f666148bd12d3034d9ddc78ff66904cfbac1aa2a24666dfb7f37d1e7R602-R609

I tried your color and see only little difference. So I changed it to yours.

Reply:
For the other themes, at first I thought it is fine to just use the default read. (as other themes are light themes). But now I write some styles for them, and I feels not bad. Have a look.
Sorry that I'm not good at styling themes. So maybe you can have a chance to continuing styling it.